src/applications/toe/components/GetHelp.jsx

```diff
@@ -6,10 +6,12 @@ export default function GetHelp() {
       <p className="help-talk">
         If you need help with your application or have questions about
         enrollment or eligibility, submit a request with{' '}
-        <a target="_blank" href="/contact-us">
+        <a target="_blank" href="https://ask.va.gov/" rel="noreferrer">
           {' '}
           Ask VA.
         </a>{' '}
+        <br />
+        <br />
```

Comment:
Maybe splitting the text into two paragraphs and using the `vads` spacing classes would be a cleaner solution?

Reply:
Got it, i'll apply the same strategy with MEB version.